src/plugins/graph/models/graph-model.ts

```diff
@@ -598,7 +642,7 @@ export const GraphModel = TileContentModel
         (ids) => {
           ids.forEach(id => {
             if (!self._idColors.has(id)) {
-              self.setColorForIdWithoutUndo(id, self.nextColor);
+              self.setColorForId(id, self.nextColor);
```

Comment:
This may require a more subtle fix, but I was getting warnings about `withoutUndo` being called as a child action.

Reply:
Yeah there is a option you pass to withoutUndo so it basically doesn't print this warning.